src/pages/iou/IOUModal.js

```diff
@@ -55,6 +55,9 @@ const propTypes = {
 
         // is loading
         loading: PropTypes.bool,
+
+        // Loading
+        isRetrievingCurrency: PropTypes.bool,
```

Comment:
I think loading can be removed from the Prop shape now.

Reply:
Also, can you make this comment a bit more descriptive? Thanks!

Reply:
Done!